main/main.c

```diff
@@ -397,7 +397,11 @@ static inline SAT_returnState init_csp_interface() {
 #endif /* CSP_USE_SDR */
 
     char rtable[128] = {0};
+#if SBAND_IS_STUBBED == 0
     snprintf(rtable, 128, "%d %s, 17 %s", gs_if_addr, gs_if_name, SDR_IF_SBAND_NAME);
+#else
+    snprintf(rtable, 128, "%d %s", gs_if_addr, gs_if_name);
```

Comment:
This string is meant to be added to depending on the way it's compiled, not the whole string defined like this

Reply:
I mean sure